src/GovUk.Education.ExploreEducationStatistics.Content.Model/Repository/MethodologyVersionRepository.cs

```diff
@@ -82,6 +82,17 @@ await _contentDbContext.Entry(methodology)
                 .ToList();
         }
 
+        public async Task<MethodologyVersion?> GetLatestPublishedVersionBySlug(string slug)
+        {
+            return await _contentDbContext
+                .MethodologyVersions
+                .Include(mv => mv.Methodology)
+                .Where(mv =>
+                    mv.Slug == slug
```

Comment:
Is it worth making this equivalence check case invariant, or do we have that implicitly by this point?

Reply:
I believe we have it implicitly.

The check is already case insensitive as in our databases:
```
SELECT 1
WHERE 'test-slug' = 'TEST-SLUG'
```
returns 1. But (in case you're not aware) the fun doesn't stop there... Our databases also don't take into account trailing whitespace in comparisons too. So:
```
SELECT 1
WHERE 'test-slug' = 'TEST-SLUG            '
```
also returns 1.

I think we have looked into changing the MSSQL defaults on this - both of these cases have stung us in the past - but for some reason buried in the past it didn't happen. Might be one worth exploring again at a dev meeting.

Other than that, I think we assume slugs are already lowercased in many places across the codebase. My understanding is that all publication and methodology slugs are currently generated via `NamingUtils.SlugFromTitle`. As far as I'm aware, the user cannot manually enter a publication/methodology slug - they can only change it indirectly by changing a title - and therefore we can currently assume that all slugs are already lowercase. Ben also did some work recently to ensure that slugs fetched from the URL are also lowercase: https://github.com/dfe-analytical-services/explore-education-statistics/pull/4322